### PR TITLE
added travis CI; please check

### PR DESCRIPTION
**Pull request by dhruvapte26**

### CODE REVIEW

**Comment by athityakumar** [comment]
I don't think that the above commit add Travis CI. Travis CI should be able to run the ruby script - to check whether the json building is successful or there are any errors.

**Comment by athityakumar** [comment]
@the-ethan-hunt : BTW, you can also add `Gemlock` file to enable `bundle install` of gems and close issue #18

**Comment by dhruvapte26** [comment]
@athityakumar Added Gemfile and Gemfile.lock. Sorry for the delay!

File: index.html [hunk 1]
```diff
@@ -392,4 +392,5 @@ <h1 class="ui inverted header">
 
 </body>
 
-</html>
\ No newline at end of file
+</html>
+  //end of page
```
Comment (athityakumar) [inline index.html]: Is this change required? Revert this.

**Review by athityakumar — CHANGES_REQUESTED** [review]

**Comment by athityakumar** [comment]
@the-ethan-hunt - Thanks for adding Gemlock files! :smile: 

Travis CI hasn't been added yet - you can add them with `travis.yml` and `Rakefile.rb`. You can get help from this repository : https://github.com/barc-iitkgp/faculty-scraper

**Comment by dhruvapte26** [comment]
updated index.html @athityakumar

**Comment by athityakumar** [comment]
Thanks for the PR! Will you be adding Travis in the same PR or later?

**Comment by dhruvapte26** [comment]
@athityakumar , CI generating error, will take some time..

**Comment by athityakumar** [comment]
Oh, okay - I'll merge this then. Add another PR for CI.

**Review by athityakumar — APPROVED** [review]

**Pull request merged**